File: DSA basic q for beg/iswordpresent.py
def iswordpresent(sentence,word):
    s = sentence.split(" ")
    for i in s:
        if i == word:
            print("yes")
            return
    print("NO")

sentence = "hello, sumit sonar here"
word = "sunit"
iswordpresent(sentence,word)

File: DSA basic q for beg/java dsa beginers/Iswordpresent.java
public class Iswordpresent {

    boolean isWordPresent(String word,String sentence){
        String[] s = sentence.split(" ");

        for(String i : s){
            if(word.equals(i)){
               return true;
            }
        }
        return false;
    }
    public static void main(String[] args) {
        String word = "hello!";
        String sentence = "hello! aman";
        Iswordpresent isw = new Iswordpresent();
        boolean ans = isw.isWordPresent(word, sentence); 
        System.out.println("answer is : "+ ans);
    }
}
